libs/gltfio/src/ResourceLoader.cpp

```diff
@@ -902,101 +897,105 @@ void ResourceLoader::updateBoundingBoxes(FFilamentAsset* asset) const {
         *result = aabb;
     };
 
-    const size_t posAttrSize = cgltf_num_components(cgltf_type_vec3);
-    const size_t skinningAttrSize = cgltf_num_components(cgltf_type_vec4);
-    auto computeBoundingBoxSkinned = [&](const cgltf_primitive* prim, const Skin* skin, Aabb* result) {
-        Aabb aabb;
-        std::vector<mat4f> inverseGlobalTransforms(skin->targets.size());
-        for (size_t i = 0; i < skin->targets.size(); i++) {
-            auto xformable = tm.getInstance(skin->targets[i]);
-            if (xformable) {
-                inverseGlobalTransforms[i] = inverse(tm.getWorldTransform(xformable));
-            }
-        }
```

Comment:
This is one of the two loops that I removed, because a given primitive can only be influenced by a single skin.